docs/user-guide/support-level.md

```diff
@@ -0,0 +1,105 @@
+# Support Level
+
+This document provides instructions on how to use the feature support-level API and how to add a new feature and set its support level dynamically.
+
+Support level is a way to determine whether a feature or architecture can be used, and it defines the current support level of that feature. 
+
+The available support-levels are:
+- **supported** - Feature is fully supported.
+- **unsupported** - Feature is not supported for the given parameters.
+- **tech-preview**: Feature is not fully supported, may not be functionally complete, and not suitable for
+  deployment in production. However, these features are provided to the customer as a courtesy and the primary goal
+  is for the feature to gain wider exposure with the goal of full support in the future.
+- **dev-preview**: Feature is considered for possible inclusion into future releases in order to collect feedback about the feature.
+
+The Support Level API has two different endpoints:
+- `GET /v2/support-levels/features` - Get the support level for a given OpenShift version and/or CPU architecture
+- `GET /v2/support-levels/architectures` - Get the supported CPU architectures for a given OpenShift version
+
+Note that the previous API `/v2/feature-support-levels` is now deprecated.
+
+
+## Feature Support Level
+This endpoint returns a list of supported features for a specific OpenShift version. 
+The support level can be evaluated by passing the CPU architecture as a query parameter.
+The result is evaluated dynamically. If a feature isn't supported on the given architecture, passing CPU architecture 
+will change only the support level and not the amount of features.
+
+
+
+### Using the Feature Support Level Endpoint
+
+To get a list of feature support levels on OpenShift version X, use the following command:
+
+```bash
+curl --request GET '<HOST>:<PORT>/api/assisted-install/v2/support-levels/features?openshift_version=<X>'
+```
+
+To get a list of feature support levels on OpenShift version X with Y CPU architecture, use the following command:
+
+
+```bash
+curl --request GET '<HOST>:<PORT>/api/assisted-install/v2/support-levels/features?openshift_version=<X>&cpu_architecture=<Y>'
+```
+
+Note that if CPU architecture is not passed as a parameter to the request, it will address the CPU architecture as x86_64.
```

Comment:
~~~suggestion
To get a list of feature support levels on OpenShift version <version> with CPU architecture <architecture>, use the following command:

```bash
curl --request GET '<HOST>:<PORT>/api/assisted-install/v2/support-levels/features?openshift_version=<version>&cpu_architecture=<architecture>'
```

Note that the CPU architecture parameter is optional and defaults to x86_64.
~~~